src/cloudFunction.ts

```diff
@@ -117,21 +118,20 @@ export class CloudFunction {
       ? opts.availableMemoryMb
       : null;
 
-    // Only one of  envVars and envVarsFile should be set
-    if (opts?.envVars && opts?.envVarsFile) {
-      throw new Error(
-        'Only one of env_vars or env_vars_file can be specified.',
-      );
-    }
+    if (opts?.envVars || opts?.envVarsFile) {
```

Comment:
nit: Can you add a comment on what's happening here?

Reply:
Done! 
thank you for the feedback!